### PR TITLE
Add BROTHERHOBBYF435

### PR DESCRIPTION
**Pull request by H-ChuWu**
Add new target BROTHERHOBBYF435

### CODE REVIEW

File: configs/BROTHERHOBBYF435/config.h [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+#pragma once
+
+#define FC_TARGET_MCU     AT32F435G
+
+#define BOARD_NAME        BROTHERHOBBYF435
+#define MANUFACTURER_ID   BROT
+
+// LEDS
+#define BEEPER_PIN             			 	PC15
```
Comment (ot0tot) [inline configs/BROTHERHOBBYF435/config.h:30]: Use spaces instead of tabs so alignment is correct

File: configs/BROTHERHOBBYF435/config.h [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+#define TIMER_PIN_MAPPING \
+ 	TIMER_PIN_MAP(0, PA8, 1, 7) \
+ 	TIMER_PIN_MAP(1, PA9, 1, 1) \
+ 	TIMER_PIN_MAP(2, PB10, 1, 13) \
+ 	TIMER_PIN_MAP(3, PB11, 1, 12) \
+ 	TIMER_PIN_MAP(4, PB7, 1, 4) \
+ 	TIMER_PIN_MAP(5, PB6, 1, 2) \
+ 	TIMER_PIN_MAP(6, PB0, 2, 9) \
+ 	TIMER_PIN_MAP(7, PB1, 2, 8) \
+ 	TIMER_PIN_MAP(8, PC9, 2, 3)
```
Comment (ot0tot) [inline configs/BROTHERHOBBYF435/config.h:134]: For MCUs with a DMA mux like the AT32F435, you can set the DMA request to 0 to let the DMA arbiter handle assignments. 
```suggestion
 	TIMER_PIN_MAP(0, PA8,  1, 0) \
 	TIMER_PIN_MAP(1, PA9,  1, 0) \
 	TIMER_PIN_MAP(2, PB10, 1, 0) \
 	TIMER_PIN_MAP(3, PB11, 1, 0) \
 	TIMER_PIN_MAP(4, PB7,  1, 0) \
 	TIMER_PIN_MAP(5, PB6,  1, 0) \
 	TIMER_PIN_MAP(6, PB0,  2, 0) \
 	TIMER_PIN_MAP(7, PB1,  2, 0) \
 	TIMER_PIN_MAP(8, PC9,  2, 0)
```

File: configs/BROTHERHOBBYF435/config.h [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+#define DEFAULT_DSHOT_BURST                 DSHOT_DMAR_AUTO
+#define DEFAULT_DSHOT_BITBANG          		DSHOT_BITBANG_ON
+#define DEFAULT_FEATURES (FEATURE_OSD | FEATURE_LED_STRIP)
```
Comment (ot0tot) [inline configs/BROTHERHOBBYF435/config.h:138]: These are not usually included in a config

File: configs/BROTHERHOBBYF435/config.h [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+// ADC			
+#define ADC_VBAT_PIN            			PC0
+#define ADC_RSSI_PIN            			PC5
+#define ADC_CURR_PIN            			PC1
+#define ADC_INSTANCE        				ADC1
+#define ADC1_DMA_OPT        				11
+#define DEFAULT_CURRENT_METER_SOURCE        CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE        VOLTAGE_METER_ADC
+#define DEFAULT_CURRENT_METER_SCALE         386
```
Comment (ot0tot) [inline configs/BROTHERHOBBYF435/config.h:117]: Is this an AIO, or does it include an ESC?

File: configs/BROTHERHOBBYF435/config.h (hunk 3 above)
Comment (H-ChuWu) [reply configs/BROTHERHOBBYF435/config.h:138]: > 这些通常不包含在配置中

So this can be deleted, right?

File: configs/BROTHERHOBBYF435/config.h (hunk 4 above)
Comment (H-ChuWu) [reply configs/BROTHERHOBBYF435/config.h:117]: > Is this an AIO, or does it include an ESC?

ESC，not AIO.

File: configs/BROTHERHOBBYF435/config.h (hunk 3 above)
Comment (sugaarK) [reply configs/BROTHERHOBBYF435/config.h]: yes

Comment (ot0tot) [reply configs/BROTHERHOBBYF435/config.h]: I was just recommending to delete `#define DEFAULT_FEATURES (FEATURE_OSD | FEATURE_LED_STRIP)`, but GitHub highlighted the previous two lines also.

**Review by ot0tot — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**